util/encoding/key_encoding_test.go

```diff
@@ -291,13 +294,76 @@ func disabledTestFloatMandE(t *testing.T) {
 		{1234.5, 2, []byte{0x19, 0x45, 0x64}},
 		{12.345, 1, []byte{0x19, 0x45, 0x64}},
 		{0.123, 0, []byte{0x19, 0x3c}},
+		{-0.123, 0, []byte{0x19, 0x3c}},
```

Comment:
Maybe add the sign bit as a third expected value in the struct; otherwise it's confusing to see 0.123 and -0.123 with the same output.